Makefile

```diff
@@ -80,28 +80,30 @@ llvm-build: llvm-build/build.ninja
 build/tinygo:
```

Comment:
TODO: add `build/lib/clang/include/stdint.h` dependency to `build/tinygo` target.